mlflow/pipelines/__init__.py

```diff
@@ -14,80 +14,16 @@
 developers a declarative approach to ML application development that reduces boilerplate.
 
 MLflow Pipelines also implements a cache-aware executor for pipeline steps, ensuring that steps
-are only executed when associated :py:ref:`code or configurations <mlflow-pipeline-layout-example>`
-have changed. This enables data scientists, ML engineers, and DevOps teams to iterate very quickly
-within their domains of expertise. MLflow offers |run() APIs| for executing pipelines, as well as
-an |mlflow pipelines run CLI|.
+are only executed when associated
+:py:ref:`code or configurations <pipeline-repositories-key-concept>` have changed. This enables
+data scientists, ML engineers, and DevOps teams to iterate very quickly within their domains of
+expertise. MLflow offers |run() APIs| for executing pipelines, as well as an
+|mlflow pipelines run CLI|.
 
-.. _mlflow-pipeline-layout-example:
```

Comment:
Moved this section out of API docs and into overview docs, since this section discusses common structures across all pipelines